eslint.config.mjs

```diff
@@ -171,6 +171,10 @@ const config = [
 
 			'@typescript-eslint/no-unsafe-call': 'off',
 			'@typescript-eslint/no-explicit-any': 'off',
+			'@typescript-eslint/no-unsafe-assignment': 'off',
+			'@typescript-eslint/no-unsafe-member-access': 'off',
+			'@typescript-eslint/no-unsafe-return': 'off',
+			'@typescript-eslint/no-unsafe-argument': 'off',
```

Comment:
Can't comment on line 182 (it's not in the PR), but I'd suggest adding `src/config.ts` to the list of (core) files to calm ESLint down about the undefined `document`.

Reply:
I wonder if we need to tweak the config somehow, I would expect `requestAnimationFrame()` would be treated as a known global.